Doc/library/webbrowser.rst

```diff
@@ -171,9 +176,6 @@ Notes:
    Removed browsers include Grail, Mosaic, Netscape, Galeon,
    Skipstone, Iceape, and Firefox versions 35 and below.
 
```

Comment:
While the MacOSX class was not explicitly documented, it was mentioned in the deprecation notice. What do you think about documenting its removal here?

Reply:
I considered it but wasn't sure, can definitely add if you like?